person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'Unknwon', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services
+    true
+  end
```

Comment:
- [ ] Please read the instructions carefully, as per the requirements, you must return a `boolean` result computed by `your` private method `of_age?`, so a person can use the services if and only if he/she at least `18` years old and have parent_permission at the same time. So you can do something like below

``` suggestion
  def can_use_services? 
    of_age?() || @parent_permission
  end

```